Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/jee/jsf/UserAuthorizationListener.java

```diff
@@ -58,20 +57,19 @@
     private static final Logger LOG = LoggerFactory.getLogger(UserAuthorizationListener.class);
 
     /**
-     * The Constant LOGIN_REQUIRED_DIR.
+     * The Constant NO_LOGIN_REQUIRED_PAGES
      */
-    public static List<String> noLoginRequiredPages = null;
+    protected static final List<String> NO_LOGIN_REQUIRED_PAGES = new ArrayList<>();
```

Comment:
This variable needs to follow constant naming conventions because it is "static final".